### DISCUSSION TITLE
Is the iam-user available to c7n-mailer?

### DISCUSSION THREAD

**Discussion by zfLQ2qx2** [cloud-custodian/cloud-custodian#7323] [General]
Quick question - particularly for events that were triggered from cloudtrail, is the username of the person who triggered the event exposed to c7n-mailer? Even if not available to c7n-mailer directly, is there a variable that I can pass through the notification subject?

I did try to read through the source before asking, I don't see a list of what interpolable variables exist, or a general way to reference items in the cloudwatch event structure from the policy statements.

**Comment by kapilt** [comment]
it is available in the mailer, for event triggered lambda policies, we serialize the event and send it along with the resource to the mailer, its available as the `event` variable in the message template.
